src/components/ReactFlagsSelect/ReactFlagsSelect.tsx

```diff
@@ -189,9 +198,17 @@ const ReactFlagsSelect: React.FC<Props> = ({
               </span>
               {showSelectedLabel && (
                 <span className={styles.label}>
-                  {getLabel(validSelectedValue)}
+                  {typeof displayLabel === "object"
+                    ? displayLabel.primary
+                    : displayLabel}
                 </span>
               )}
+              {showSecondarySelectedLabel &&
+                typeof displayLabel === "object" && (
+                  <span className={[styles.label, styles.secondary].join(" ")}>
```

Comment:
we are using `classnames` library to merge classes in this project.
`cx(styles.label, styles.secondaryLabel)`